public/static/styles/app.css

```
@@ -594,7 +480,7 @@
   }
   body {
     @apply h-full overflow-y-scroll bg-background text-foreground;
-    font-family: 'Inter', sans-serif;
+    font-family: var(--font-inter, sans-serif);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)